cardinal/ecs/storage/redis_storage.go

```diff
@@ -46,11 +47,32 @@ func NewRedisStorage(options Options, namespace string) RedisStorage {
 	}
 }
 
+// ---------------------------------------------------------------------------
```

Comment:
Sorry if it was unclear before, what i mean is that this entire block of "Schema Storage" should just go into `/storage/redis/schema.go`